### PR TITLE
GDB-9610: Document version 11

### PR DESCRIPTION
**Pull request by mihailradkov**
- Revised the readme file
- Changelog updates
- Updated the UPGRADE.md with migration notes for version 11.
- Added NGINX examples

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+The downside of this option is that if there are a lot of GBs of data to be backed up and later restored, this option would be the slowest.
+
+**Matching PVC claims**
+
+To minimize the downtime from **Backup and Restore**, you could reuse the existing Persistent Volumes from the deployment made with version 10 of 
+the chart. The procedure is as follows:
+
+* Make note of the names of the existing PVs and PVCs:
+  * Kubernetes uses the following policy for naming PVC: `<pvc-template-name>-<statefulset-name>-<pod index>`, i.e. this would be
```
Comment (mihailradkov) [inline UPGRADE.md:96]: Use numbered lists

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+#### Fixing Permissions
+
+Version 11 of the chart includes a default security context that avoids running containers with the `root` user. 
+However, version 10 of the chart have been running containers with the `root` user.
+This requires changing ownership of the data when using existing Persistent Volumes. 
+
+Note that if you've selected to use the **Backup and Restore** option for the data migration, these next steps are not needed.
+
+There are 2 options that you can choose from:
```
Comment (mihailradkov) [inline UPGRADE.md:137]: Could mention a 3rd: manually doing this by mounting the volume

File: UPGRADE.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+We strongly advise to operate a Kubernetes cluster with version that has not reached its End Of Life. 
+
+**Naming**
+
+The resource names are no longer hardcoded in version 11. If you want to keep the old ones, you can override the following configurations from 
+[values.yaml](values.yaml): `fullnameOverride` and `proxy.fullnameOverride`.
+
+**GraphDB URL**
+
+The old version 11 of the chart had several configuration properties that defined the external URL of GraphDB: `deployment.protocol`,
```
Comment (viktor-ribchev) [inline UPGRADE.md:26]: The old version 11

File: UPGRADE.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+**Storage Class**
+
+Version 11 removes the hardcoded storage class from `global.storageClass`.
+If you don't have a default storage class in your cluster, you can define the storage class for GraphDB's PVC
+with `persistence.volumeClaimTemplate.spec.storageClassName` for GraphDB and `persistence.volumeClaimTemplate.spec.storageClassName` for the proxy (if
+enabled).
+
+**Persistence**
+
+Persistence configurations have been moved to `persistence` and `proxy.persistence`.
```
Comment (viktor-ribchev) [inline UPGRADE.md:50]: This is a bit confusing, all persistence configurations?

File: UPGRADE.md (hunk 3 above)
Comment (mihailradkov) [reply UPGRADE.md:26]: Ooops

File: UPGRADE.md (hunk 4 above)
Comment (mihailradkov) [reply UPGRADE.md:50]: I'll mention the previous location

File: UPGRADE.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+
+Note that all security configurations have been moved under the `security` section in [values.yaml](values.yaml).
+
+The provisioning user credentials are now under `security.provisioner`.
+
+**Other**
+
+See [11.0.0 Breaking Changes section](./CHANGELOG.md) for more details on migrating other configurations.
+
+### Data Migration
```
Comment (mihailradkov) [inline UPGRADE.md:72]: A 3rd option exists: manual cluster recovery.

**Pull request merged**